go.mod

```diff
@@ -1,13 +1,13 @@
 module github.com/vmware-tanzu/velero
 
-go 1.23.0
+go 1.23.1
```

Comment:
This is coming from https://github.com/kubernetes-sigs/cloud-provider-azure/commit/5548c42f4827079a5127c6f41f2e055a4d076878 ("sigs.k8s.io/cloud-provider-azure/pkg/azclient")

Filed issue there to hopefully [starve off](https://github.com/kubernetes-sigs/cloud-provider-azure/issues/8535) future unnecessary bumps.